qiskit_nature/problems/second_quantization/vibrational/vibrational_op_builder.py

```diff
@@ -9,38 +9,40 @@
 # Any modifications or derivative works of this code must retain this
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
-""" Vibrational Spin operator builder. """
+""" Vibrational operator builder. """
 from typing import Union, List
 
 from qiskit_nature.drivers import WatsonHamiltonian
 from qiskit_nature.drivers.bosonic_bases import HarmonicBasis
 
-from qiskit_nature.operators.second_quantization.vibrational_spin_op import VibrationalSpinOp
+from qiskit_nature.operators.second_quantization.vibrational_op import VibrationalOp
 from qiskit_nature.problems.second_quantization.vibrational.vibrational_label_builder import \
     _create_labels
 
 
-def build_vibrational_spin_op(watson_hamiltonian: WatsonHamiltonian,
-                              basis_size: Union[int, List[int]],
-                              truncation_order: int):
+def build_vibrational_op(watson_hamiltonian: WatsonHamiltonian,
+                         num_modals: Union[int, List[int]],
+                         truncation_order: int) -> VibrationalOp:
     """
-    Builds a spin operator based on a WatsonHamiltonian object.
+    Builds a VibrationalOp based on a WatsonHamiltonian object.
 
     Args:
         watson_hamiltonian (WatsonHamiltonian): WatsonHamiltonian instance.
-        basis_size: size of a basis
+        num_modals: the number of modals per mode.
         truncation_order: order at which an n-body expansion is truncated
 
     Returns:
-        SpinOp: SpinOp built from a WatsonHamiltonian object.
+        VibrationalOp: VibrationalOp built from a WatsonHamiltonian object.
     """
-
     num_modes = watson_hamiltonian.num_modes
-    if isinstance(basis_size, int):
-        basis_size = [basis_size] * num_modes
-    # TODO make HarmonicBasis an argument and support other bases when implemented
+
+    if isinstance(num_modals, int):
+        num_modals = [num_modals] * num_modes
+
+    # TODO: make HarmonicBasis an argument and support other bases when implemented
```

Comment:
I guess if this argument is added on the end in the future and defaults to HarmonicBasis internally then things would stay compatible. Just a comment against the TODO which I am assuming the intent is that this would not be done this release